### DISCUSSION TITLE
[Feature]: Nim lang support

### DISCUSSION THREAD

**Discussion by xTrayambak** [ROCm/ROCm#3487] [Ideas]
### Suggestion Description

It would be nice if ROCm had Nim bindings as it's really a joy to write code in it whilst still maintaining good performance. It shouldn't be that hard either since Nim has some of the best FFI you can ask for (it compiles to C or C++ so you can easily import stuff from C/C++ land). I can help create some bindings if needed.

**Comment by ppanchad-amd** [comment]
@xTrayambak Please open this under 'Discussions' section. Closing ticket under 'Issues' section. Thanks!

**Reply by saadrahim** [reply]
I converted this issue to a discussion.
